common/src/main/java/com/wynntils/mc/event/PlayerInteractEvent.java

```diff
@@ -84,9 +84,9 @@ public BlockPos getPos() {
         }
 
         @Override
-        public void setCanceled(boolean canceled) {
-            super.setCanceled(canceled);
-            if (canceled) {
+        public void setCanceled(boolean cancel) {
+            super.setCanceled(cancel);
+            if (cancel) {
```

Comment:
I doubt this is an intellij sugestion?

Reply:
The parameter is overridden. The name is from the parent

Reply:
Okay, fine, keep this one.

Reply:
Actually, I disagree. That is a particularly stupid suggestion from IntelliJ. We should turn it off in the Artemis analysis profile instead. 

The overriding method has a lot more special knowledge, that might warrant a different name.

And methods overridden from Vanilla will often have bad argument names due to the deobfuscation.

Reply:
so should i change it back?

Reply:
yes